pandas/io/json/_normalize.py

```diff
@@ -207,6 +261,23 @@ def json_normalize(
     1   130              60          NaN    Mose Reg
     2   130              60          2.0    Faye Raker
 
+    Normalizes nested data up to level 1.
```

Comment:
can you add an example with a str / list-of-str? are these useful? better to just make this a callable only?

Reply:
The initial proposal was to provide a list of keys to ignore. After a discussion with @WillAyd he suggested list and list of str be consistent with the inclusion in other modules which made sense. 

With callable only I'm not sure if we can achieve the multi-level support we are talking about iin 
https://github.com/pandas-dev/pandas/pull/27262#issuecomment-524528962

Let me know your thoughts